index.js

```diff
@@ -0,0 +1,19 @@
+const app = require('./app')
+const port = process.env.PORT || 8080
```

Comment:
Would be great to have a config file for this - check this from last year: https://github.com/Rostlab/JS16_ProjectA/tree/master/cfg